### PR TITLE
Add act under new section named Tools

### PR DESCRIPTION
**Pull request by cplee**
* Create new section `Tools`
* Add `act` under new `Tools` section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -408,6 +409,10 @@ Set up your GitHub Actions workflow with a specific version of your programming
 
 - [GitHub Actions Branding Cheat Sheet](https://haya14busa.github.io/github-action-brandings/)
 
+### Tools
+
+- [Run GitHub Actions Locally](https://github.com/nektos/act)
```
Comment (jessicalostinspace) [inline README.md]: It looks like this action may already be in the GitHub Tools and Management Section under "Run GitHub Actions Locally with a web interface". Would you mind providing additional info on how this action is different and moving it to that section?

Comment (cplee) [reply README.md]: @jessicalostinspace - i moved it under that same section.  fwiw, I originally added this tool over a year ago in #53

**Comment by jessicalostinspace** [comment]
I actually think there is a difference between the two actions (the one I mentioned previously vs. this one) and value to having both. Sorry for the inconvenience of update + failure of linter, there was an issue that I think is now fixed in this repo - you may have to pull latest. Could you add yours right underneath the other Action with the title "Run GitHub Actions Locally with a web interface" and name it "Run GitHub Actions Locally in Terminal"

**Comment by cplee** [comment]
Thanks @jessicalostinspace - updates made

**Pull request merged**